fmriprep/workflows/anatomical.py

```diff
@@ -894,25 +900,21 @@ def init_anat_reports_wf(reportlets_dir, output_spaces,
 
     inputnode = pe.Node(
         niu.IdentityInterface(
-            fields=['source_file', 't1_conform_report', 't1_seg_report',
-                    't1_2_mni_report', 't1_skull_strip_report', 'recon_report']),
+            fields=['source_file', 't1_conform_report', 'seg_report',
+                    't1_2_mni_report', 'recon_report']),
         name='inputnode')
 
     ds_t1_conform_report = pe.Node(
         DerivativesDataSink(base_directory=reportlets_dir, suffix='conform'),
         name='ds_t1_conform_report', run_without_submitting=True)
 
-    ds_t1_seg_report = pe.Node(
-        DerivativesDataSink(base_directory=reportlets_dir, suffix='t1_seg'),
-        name='ds_t1_seg_report', run_without_submitting=True)
-
     ds_t1_2_mni_report = pe.Node(
         DerivativesDataSink(base_directory=reportlets_dir, suffix='t1_2_mni'),
         name='ds_t1_2_mni_report', run_without_submitting=True)
 
-    ds_t1_skull_strip_report = pe.Node(
-        DerivativesDataSink(base_directory=reportlets_dir, suffix='t1_skull_strip'),
-        name='ds_t1_skull_strip_report', run_without_submitting=True)
+    ds_seg_brainmask = pe.Node(
```

Comment:
For consistency and clarity, I'd prefer to keep `t1` and `report` in the name until/unless we decide to change all these nodes. Maybe `ds_t1_seg_mask_report` or similar?